content/blog/2022-01-31-january-22-community-gems.md

```diff
@@ -0,0 +1,191 @@
+        mode='rb'
+        )
+    )
+```
+
+### [One of the steps in my DVC pipeline uses a `pip` installed package. What is the best way to make sure that DVC re-runs the steps that depend on that package?](https://discord.com/channels/485586884165107732/563406153334128681/920139825284280381)
+
+Thanks for the question @alphaomega!
+
+The best way to handle any package dependencies will be to include a
+requirements.txt file with the specific versions your pipeline needs.
+
+Another approach you can take is having a stage that dumps the package version
+as an intermediate output. It doesn't have to be saved in Git or DVC becasue
+it's easily reproduced and the DVC internals should be able to take care of
+detecting that the package didn't change.
+
+### [Does DVC track dependencies which are in the `dvc.yaml` pipeline?](https://discord.com/channels/485586884165107732/563406153334128681/920659549835370497)
+
+Thanks for another great question @rie!
+
+DVC doesn't track the pipeline dependencies, only the outputs. If you want DVC
+to track a pure data dependency that's not an output of a different stage, you
+need to track it with `dvc add ...`
+
+### [What is the difference between Kubeflow pipelines and DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/922728960478035978)
+
+This is a fantastic question! Thanks for asking @ramakrishnamamidi!
+
+A major difference is that DVC focuses primarily on ML _development_ and adding
+lightweight functionality on top of existing projects, which might be reusable
+in deployment in some cases.
+
+Kubeflow focuses on _deployment_ and building on top of Kuberenetes, which could
+be used during development but requires more up-front effort.
+
+### [Could DVC be a good alternative to LFS for game development?](https://discord.com/channels/485586884165107732/485586884165107734/928336349487067196)
+
+Thanks for such an interesting question @CB!
+
+Yes! We have community members that use DVC to handle their large files in game
+development.
+
+There are several other use cases we've seen for DVC outside of machine learning
+and data science. Some users have used DVC to track build artifacts for
+deployment systems and to track performance data alongside design iterations and
+simulation tools.
+
+### [Does DVC run on JSON/YAML configuration files for all things?](https://discord.com/channels/485586884165107732/563406153334128681/928779586622332938)
+
+This is a great question about large projects with a lot of dependencies from
+@SolemnSimulacrum!
+
+All of the dependencies you list in `dvc run` are in fact configured in the
+`dvc.yaml` file. `dvc run` is a convenience for adding a pipeline stage to this
+file and then doing `dvc repro` on that stage. It's completely acceptable and
+even encouraged to directly edit `dvc.yaml` if that's easier.
+
+For example, if you are currently executing a command like this:
+
+```dvc
```

Comment:
Can you make the `dvc run` and `dvc.yaml` examples here consistent with each other? The dependencies line up but otherwise there are a lot of differences (stage name, command, parameters, outputs, metrics, plots).

Also, could you remove `-f` here? It's irrelevant to the question, and it might add confusion.